package.json

```diff
@@ -30,10 +30,14 @@
     "build:all": "cd packages/app && npm run build:dist && npm run build:electron && npm run build:all"
   },
   "dependencies": {
-    "eslint": "^6.5.1"
+    "eslint": "^6.5.1",
+    "tiny-async-pool": "^1.0.4"
```

Comment:
Please add dependencies to specific packages. The root package.json should only have dependencies that are needed to run lerna.